ko/sql/tuning.rst

```diff
@@ -3831,6 +3831,161 @@ N:1 관계의 **LEFT OUTER JOIN**\에서 조인 조건 외에 오른쪽 테이
                 9           60          600  'Left- 806'                     6
                10           60          600  'Left- 906'                     6
 
+.. _view_merge:
+
+View Merging 최적화
+-----------------------
+View Merging은 쿼리 처리 시간과 오버헤드를 줄이는데 초점을 둔다. 쿼리가 뷰를 사용할 때, 시스템은 일반적으로 새로운 임시 테이블을 생성한다. 
+
+그러나 이렇게 생성된 임시 테이블은 인덱스를 사용하기 어렵고, 뷰를 생성하는 과정 자체가 시스템에 불필요한 부하를 준다. 
+
+따라서 View Merging은 뷰를 원래 테이블로 대체하여 이러한 오버헤드를 피하고, 원래 테이블의
+인덱스를 활용하여 더 효율적인 쿼리 처리를 가능하게 한다.
```

Comment:
It is trivial.

How about writing it on a single line with no line breaks?

Reply:
I think it's a good suggestion, so I've incorporated it.